pkg/ali/util/ack.ts

```diff
@@ -34,6 +34,118 @@ async function getACKOptions(
   return store.dispatch("cluster/request", { url });
 }
 
+async function getPaginatedACKOptions(store: any,
```

Comment:
nit: a large amount of the code between `getPaginatedACKOptions` & `getPaginatedClusters` is duplicated, differing mainly in the format of the response. Consider refactoring in the future to consolidate.